api/src/repositories/observation-repository.ts

```diff
@@ -211,6 +211,26 @@ export class ObservationRepository extends BaseRepository {
     return response.rows;
   }
 
+  /**
+   * Retrieves the count of survey observations for the given survey
+   *
+   * @param {number} surveyId
+   * @return {*}  {Promise<{ observationCount: number }>}
+   * @memberof ObservationRepository
+   */
+  async getSurveyObservationCount(surveyId: number): Promise<{ observationCount: number }> {
+    const knex = getKnex();
+    const sqlStatement = knex
+      .queryBuilder()
+      .count('survey_observation_id as rowCount')
+      .from('survey_observation')
+      .where('survey_id', surveyId);
+
+    const response = await this.connection.knex(sqlStatement);
+    const observationCount = parseInt(response.rows[0].rowCount, 10);
```

Comment:
Why `parseInt` instead of `Number`?

Reply:
Yeah good point, I can't think of a valid counter-argument. I think number is preferable here.